index-es5.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <title>Chat Adapter Prototype</title>
+  <script src="http://localhost:8080/dist/chat-adapter.es5.js"></script>
+  <script src="https://cdn.botframework.com/botframework-webchat/4.8.0/webchat-es5.js"></script>
```

Comment:
I think Web Chat doesn't upload new builds to cdn and if we want to use their alpha versions we either need to upload by ourselves or use npm